src/utils.ts

```diff
@@ -142,11 +144,16 @@ export async function buildCeramic (configObj, ipfs?: IpfsApi): Promise<CeramicA
  * restarting `ceramic` is supported, restarting `ceramicClient` is not.  This is because
  * `ceramicClient` is always expected to be in `client` mode.
  */
-export async function restartCeramic() {
+export async function restartCeramic(): Promise<void> {
     if (config.jest.services.ceramic.mode == "client") {
         throw new Error("Cannot restart ceramic node running in http client mode")
     }
     await ceramic.close()
+    await delay(3000) // Give some time for things to fully shut down before restarting
```

Comment:
Can do something similar to the bash ready-wait loop above to detect whether all services have completely shut down.

Reply:
Another reason to do this is that failures in groups of tests can often be because of earlier tests trampling later ones - ensuring clean startup and shutdown from test to test is a good way to avoid that.